portfolio/src/main/webapp/assets/script.js

```diff
@@ -130,17 +101,43 @@ function scrollUpHelper() {
     var top = document.getElementById("top-button");
     if (document.documentElement.scrollTop > 20) {
         top.style.display = "block";
-    } else {
+    } else if (top) {
```

Comment:
Why check for existence here?

Reply:
When I was checking the console.log output, it gave an element not found error without checking for existence -- I wasn't sure if that was an issue so I checked for existence.